src/collective/frontpage/browser/cache.zcml

```diff
@@ -0,0 +1,19 @@
+<configure
```

Comment:
This is to configure a caching server in front of Zope. Has this any effect on your local system? I'd prefer some widely used cache decorators.

Reply:
Correct, although it does set a caching header in the page response. So when running the server (not in foreground) there are some benefits to this.

But I have replaced it with the view memoize feature.